doc/source/notebooks/tailor/mixture_density_network.ipynb

```diff
@@ -202,9 +202,9 @@
     "$$\n",
     "\\sum_{m=1}^{M} \\pi_{m}(x) = 1, \\pi_m \\ge 0\\ \\text{and}\\ \\sigma_m\\ \\forall\\ m\n",
     "$$\n",
-    "We achieve this by applying the `softmax` operator to the $\\pi$'s and by taking the `exp` to the $\\sigma$'s.\n",
+    "We achieve this by applying the `softmax` operator to the $\\pi$s and by taking the `exp` to the $\\sigma$s.\n",
```

Comment:
I tend to be in favour of apostrophes to make plural-symbols to distinguish it from multiplying \pi or \sigma by the variable s...